README.adoc

```diff
@@ -26,11 +26,18 @@ Running on the following CI services:
 * https://github.com/sonarsource-cfamily-examples?q=jenkins[Jenkins]
 * https://github.com/sonarsource-cfamily-examples?q=gitlab[GitLab CI]
 * https://github.com/sonarsource-cfamily-examples?q=bitbucket[BitBucket Pipelines]
+* plus https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=otherci[generic CI]
+* and https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=manual[manually]
```

Comment:
```suggestion
Additionally, you can find [generic examples](https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=otherci[generic examples]) that work for other CIs as well as [manual configurations](https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=manual[manual configurations]) of SonarQube.
```

Reply:
I don't find the fact of adding these two items to the list very clear. I'd instead write them down in a sentence like the one above. WDYT?

Reply:
I think adding a top-level entry like this breaks the flow of the description. I tried to clarify it, PTAL.

Reply:
Perfect!